src/Compilers/CSharp/csc/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.DiaSymReader.Native": "1.5.0"
+    "Microsoft.DiaSymReader.Native": "1.6.0-beta2-25219"
```

Comment:
I should probably clarify. The flow for this package is documented here: https://github.com/dotnet/roslyn-debug/tree/master/src/Microsoft.DiaSymReader.Native. To get an RTM binaries we need to wait until VS RTM build is produced by the lab.